### PR TITLE
Translation to Bahasa Melayu

### PR DESCRIPTION
**Pull request by alkaha**
Hi,
I've started some translations to Bahasa Melayu(Malay).  Completed the 5 first lessons under basics folder.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -16,10 +16,11 @@ description:
   pt: Lições sobre a linguagem de programação Elixir
   vi: Lessons about the Elixir programming language
   ru: Lessons about the Elixir programming language
+  my: Pelajaran mengenai bahasa aturcara Elixir
 baseurl: /
-url: https://elixirschool.com
+url: http://localhost:4000
```
Comment (doomspork) [inline _config.yml]: Please undo this change

File: _config.yml [hunk 2]
```diff
@@ -16,10 +16,11 @@ description:
...
 permalink: pretty
-languages: ['en', 'jp', 'cn', 'es', 'pt', 'vi', 'ru']
+languages: ['en', 'jp', 'cn', 'es', 'pt', 'vi', 'ru', 'my']
```
Comment (doomspork) [inline _config.yml]: You need to rebase, there are more languages now.

**Comment by doomspork** [comment]
This is great @alkaha!  I left some feedback for you.

**Comment by doomspork** [comment]
Awesome @alkaha!  What is your twitter handle btw?  When I merge these I'd like to shoot out a tweet and make sure you get credit :grinning:

**Comment by doomspork** [comment]
One last suggestion @alkaha, would you be open to adding Bahasa Melayu to the `index.md`?  We haven't been good about making sure each new language is added, see here: https://github.com/alkaha/elixir-school/blob/master/index.md

**Comment by alkaha** [comment]
Glad to help. :)
I'm @alkaha on twitter and I'll edit the index.md to add Bahasa Melayu.

File: my/lessons/basics/enum.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
+---
+layout: page
+title: Enum
+category: basics
+order: 3
+lang: en
```
Comment (doomspork) [inline my/lessons/basics/enum.md]: For the time being can you remove these `en` lessons or change the lang to `my`?  Unfortunately this way will cause issues with the site.

**Comment by alkaha** [comment]
I removed the files mentioned.
sorry about that.

**Comment by doomspork** [comment]
Thank you again @alkaha!  

I will create an Issue in to help manage the remaining lessons.
